src/app/drive/components/DriveItemDropdownActions/DriveItemDropdownActions.tsx

```diff
@@ -96,7 +96,7 @@ const FileDropdownActions = (props: FileDropdownActionsProps) => {
       {openDropdown && item && (
         <>
           {menuItems()?.map((option, i) => (
-            <div key={option?.name}>
+            <div key={i}>
```

Comment:
Do not use array indexes on the keys of a list, this negatively affects rendering performance. Use unique identifiers that do not change when identifying each element

Reply:
I did it temporally because I had to migrate this contextmenu, but I forgot. Working on it!